blog/fine-grained-news-2024-03.md

```diff
@@ -0,0 +1,77 @@
+## New Adopters
+
+The OpenFGA community [maintains a list](https://github.com/openfga/community/blob/main/ADOPTERS.md) of products/projects/companies that are using OpenFGA in production. We'd like to thank thank the following adopters for adding themselves to the list in the last month:
+
+- [Instill AI](https://www.instill.tech/)
+- [Zuplo](https://zuplo.com)
+- [OpenObserve](https://openobserve.ai/)
+- [Datum](https://datum.net/)
+
+If you are using OpenFGA in production, please consider adding your company/project to the list.
+ 
+ ## Community News
+
+- [Raghd Hamzeh](https://github.com/rhamzeh/) represented OpenFGA in an [episode on Authorizing Access](https://www.youtube.com/watch?v=VyHbFwfrf04) within a series called "You Choose - Choose Your Own Adventure: The Treacherous Trek to Security”. This episode was comparing OpenFGA with Hexa and Paralus. OpenFGA was the project viewers voted for as most interested in being featured in a demo.
+
+- [Sam Bellen](https://github.com/Sambego) published a [Google Drive example](https://github.com/Sambego/fga-drive-example/) using OpenFGA. It's a Next.js project, written in TypeScript and ready to deploy on Vercel.
+
+- [Philipp Wagner](https://github.com/bytefish) is working on a [.NET example](https://github.com/bytefish/gitclub-dotnet) inspired by the Github model.
+
+- [Pauline Jamin](https://github.com/paulinejamin) and [Geoffroy Braun](https://github.com/geoffroybraun) will present about [Infuser du métier dans les autorisations avec ReBAC](
+https://www.devoxx.fr/schedule/talk/?id=42356) at [Devoxx France 2024](https://www.devoxx.fr/) in April 17th.
+
+## New Releases
+
+We just shipped a [release candidate of Modular Models](https://openfga.dev/blog/modular-models-announcement), that makes it easy for multiple teams to collaborate in a single OpenFGA model. It requires the following components:
+
+- [OpenFGA v.1.5.1](https://github.com/openfga/js-sdk/releases/tag/v0.3.5) 
+- [CLI v0.3.0](https://github.com/openfga/cli/releases/tag/v0.3.0) 
+- [Visual Studio Code Extension v0.2.20](https://github.com/openfga/vscode-ext/releases/tag/v0.2.20)
+
+We also shipped new version of our SDKs with several fixes:
+
+- [Javascript SDK 0.3.5](https://github.com/openfga/js-sdk/releases/tag/v0.3.5).
+- [Go SDK v0.3.5](https://github.com/openfga/go-sdk/releases/tag/v0.3.5)
+- [Java SDK v0.4.0](https://github.com/openfga/java-sdk/releases/tag/v0.4.0)
+
+## Transitioning from Discord to CNCF's Slack
+
+As we mentioned in the last edition, we transitioned out from Discord for OpenFGA and are now using the CNCF [OpenFGA Slack channel](https://cloud-native.slack.com/archives/C06G1NNH47N). If you are not part of the CNCF Slack workspace, you need to join the [CNCF Slack](https://slack.cncf.io) first.
+
+## See you next month!
+
+Fine Grained News are published every month. If you have any feedback, you want to share your OpenFGA story, or know about something that you think is worth mentioning, please let us know!
```

Comment:
If you have any feedback, ~~you~~ want to share your OpenFGA story, or know about something